.changeset/sweet-clocks-shave.md

```diff
@@ -0,0 +1,5 @@
+---
+"ledger-live-desktop": minor
```

Comment:
Super unimportant but I think these changes are more of a `patch`

Reply:
:ah: you're probably right